pbm/client.go

```diff
@@ -94,3 +97,121 @@ func (c *Client) CheckRequirements(ctx context.Context, hubs []types.PbmPlacemen
 
 	return res.Returnval, nil
 }
+
+func (l PlacementCompatibilityResult) GetCompatibleDatastores() []types.PbmPlacementHub {
```

Comment:
Nit: remove "Get" prefix from this and the method below.

See: https://golang.org/doc/effective_go.html#Getters